client/src/routes/Pages/FourOFour/FourOFour.js

```diff
@@ -5,10 +5,10 @@ import './FourOFour.scss';
 
 const FourOFour = () => (
   <main class="four-o-four">
-    <img src={FourOFourVis} />
+    <img src={FourOFourVis} alt="Powered by DigitalOcean" />
```

Comment:
It was required

Reply:
Ok, then "Powered by DigitalOcean" does not make sense for this image. It should be changed to "404 error image" I guess.